admin/common_auth/views.py

```diff
@@ -21,6 +24,47 @@ def login(request):
     context = {'form': form}
     return render(request, 'login.html', context)
 
+
 def logout(request):
     logout_user(request)
     return redirect('auth:login')
+
+
+# Permission restriction based on staff (only staff can register new users). Use is_staff boolean?
+def register(request):
+    if request.user.is_staff:
+        if request.method == 'POST':
+            form = CustomUserRegistrationForm(request.POST)
+            if form.is_valid():
+                email = form.cleaned_data['email']
+                password = form.cleaned_data['password1']
+                new_user = MyUser.objects.create_user(email=email, password=password)
+                new_user.first_name = form.cleaned_data['first_name']
+                new_user.last_name = form.cleaned_data['last_name']
+                group_perms = form.cleaned_data['group_perms']
+
+                for group in group_perms:
+                    new_user.groups.add(group)
```

Comment:
I added a group, and I still get an empty permissions list on the user registration page.

![screen shot 2016-03-21 at 4 18 51 pm](https://cloud.githubusercontent.com/assets/134455/13932874/ae1f77f4-ef80-11e5-9947-4f0627f80564.png)

![screen shot 2016-03-21 at 4 19 33 pm](https://cloud.githubusercontent.com/assets/134455/13932883/b987f01c-ef80-11e5-95d8-f37b8309c859.png)